samples/misc/nif/nif.c

```diff
@@ -122,9 +127,17 @@ int main(int argc, char* argv[])
 	NOTEID testid = 0x123;                       /* test if dor notesin view test */
 	BOOL bResult = FALSE;                        /* To store the bool output */
 	NOTEID TimeVariantViewid;                    /* id of the Time variant view from db*/
+	char szJSON[STRING_LENGTH] = {0};
+	BOOL bJsonEnabled = FALSE;
 	
-	ProcessArgs(argc, argv, szDbName, szTimeVariantView, szViewName);
-	
+	ProcessArgs(argc, argv, szDbName, szTimeVariantView, szViewName, szJSON);
+
+	if (!strcmp(argv[4], "json")) {

```

Comment:
Not clear on what is the purpose of szJSON variable as you use argv[4] in if block.